types/dragula/v2/index.d.ts

```diff
@@ -17,6 +17,8 @@ declare namespace dragula {
         delay?: boolean | number | undefined;
         mirrorContainer?: Element | undefined;
         ignoreInputTextSelection?: boolean | undefined;
+        slideFactorX?: number | undefined;
+        slideFactorY?: number | undefined;
```

Comment:
It doesn’t look like these exist in v2: https://github.com/bevacqua/dragula/tree/v2.1.2

Reply:
removed